velox/common/encode/Base64.cpp

```diff
@@ -433,7 +433,20 @@ std::string Base64::encodeUrl(const folly::IOBuf* data) {
   return encodeImpl(IOBufWrapper(data), kBase64UrlCharset, false);
 }
 
+<<<<<<< HEAD
```

Comment:
There are still some conflicts

Reply:
@Yuhta my apology, IDE caused some issues, showed the conflict is gone but internally it stays so had to do it on the terminal. I have squashed and pushed, should be good now, thanks!